public/index.html

```diff
@@ -23,21 +23,25 @@
 
     <!-- Navbar -->
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
-      <a class="navbar-brand" href="/"><img src="/assets/icon.png" class="img-fluid"></a>
+      <a class="navbar-brand" href="/"><img src="/assets/icon.png" class="img-fluid" width="70%"></a>
       <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
     
       <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav ml-auto">
-          <li class="nav-item">
-            <a class="nav-link" href="/articles">Articles</a>
-          </li>
-
-          <li class="nav-item">
-            <a class="nav-link" href="/tags">Tags</a>
-          </li>
-        </ul>
+          <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="/articles">Articles</a>
```

Comment:
is there a vue.js method for routing instead of hardcoding?

Reply:
Navbar is outside the VueJS div... So vue-router links will not work here